src/kitovu/sync/syncing.py

```diff
@@ -10,6 +10,7 @@
 from kitovu.sync import syncplugin
 from kitovu.sync.settings import Settings, ConnectionSettings
 from kitovu.sync.plugin import smb
+from kitovu.sync import kitovu_filecache
```

Comment:
Why is the file named `kitovu_filecache` by the way? For consistency, you should drop that prefix. Make sure to rename it in a git commit without any other changes though, that makes things easier to follow.